operations/system_management_health/uan_node_exporter_configs.md

```diff
@@ -156,10 +156,10 @@ This procedure is to configure the UAN NODE Exporter once the PIT node no longer
 1. (`ncn#`) Redeploy the same chart version but with the desired UAN NODE Exporter configuration settings.
 
    ```bash
-   loftsman ship charts-path /helm --manifest-path /root/manifest.yaml
+   loftsman ship --charts-path /etc/cray/upgrade/csm/csm-1.4.3/tarball/csm-1.4.3/helm/ --manifest-path manifest.yaml
    ```
 
-   Here, /helm is the path of the `cray-sysmgmt-health` chart.
+   Here, `/etc/cray/upgrade/csm/csm-1.4.3/tarball/csm-1.4.3/helm/` is the path of the `cray-sysmgmt-health` chart.
```

Comment:
This should use the `CSM_RELEASE` variable in `/etc/environment` instead of `1.4.3`.